pkgs/by-name/mr/mricron/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  atk,
+  autoPatchelfHook,
+  cairo,
+  copyDesktopItems,
+  freetype,
+  fontconfig,
+  lib,
+  stdenv,
+  fetchurl,
+  gtk2,
+  glib,
+  gdk-pixbuf,
+  makeWrapper,
+  makeDesktopItem,
+  pango,
+  unzip,
+  xorg,
+  zlib,
+}:
+stdenv.mkDerivation rec {
+  pname = "mricron";
+  version = "1.0.20190902";
+
+  src = fetchurl {
+    url = "https://github.com/neurolabusc/MRIcron/releases/download/v${version}/MRIcron_linux.zip";
+    hash = "sha256-C155u9dvYEyWRfTv3KNQFI6aMWIAjgvdSIqMuYVIOQA=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+    makeWrapper
+    unzip
+  ];
+  buildInputs = [
+    atk
+    cairo
+    freetype
+    fontconfig
+    gtk2
+    glib
+    gdk-pixbuf
+    pango
+    xorg.libX11
+    zlib
+  ];
```

Comment:
```suggestion

  buildInputs = [
    atk
    cairo
    freetype
    fontconfig
    gtk2
    glib
    gdk-pixbuf
    pango
    xorg.libX11
    zlib
  ];

```

Reply:
hi @SuperSandro2000 I cannot see a difference here. Got something lost during the force-pushs, or am I blind 😆

Reply:
ah my fault. Yes seems so.

Reply:
Now It should be ok?

Reply:
Is there not just a couple newlines at beginning and end?

Reply:
yes, I think so. But it doesn't make sense, as there are spaces above and below. I suspect the format is now as it should be. 
If Sandro doesn't protest, I would just resolve this comment